site/deploy.md

```diff
@@ -28,6 +28,7 @@ When running the tool in WLST online mode, the deploy operation may require serv
 - `101` - The domain needs to be restarted and the Deploy Applications Tool needs to be re-invoked with the same arguments.
```

Comment:
101 and 102 are not valid are they? the rollback command line arg is not documented.

Reply:
I don't see anywhere returning 101, 102 from the source. I am removing them now.